packages/component_code_gen/code_gen/generate_for_github_issue.py

```diff
@@ -180,9 +180,9 @@ def generate(issue_number, output_dir, generate_pr=True, clean=False, verbose=Fa
         # XXX: remove ts stuff and .gitignore
         # GitPython requires to manually check .gitignore paths when adding files to index
         # so this is easier
-        run_command(f"npx eslint {app_base_path} --fix")
+        run_command(f"npx eslint {app_base_path} --fix --ignore-pattern node_modules")
         run_command(["npx pnpm i"])
```

Comment:
Can you check if the `[]` is necessary here?

```suggestion
        run_command(["npx pnpm i"])
```